examples/example_omb_csv_reader.py

```diff
@@ -0,0 +1,53 @@
+"""
+Reading an OMB Rock7 CSV file into trajan
+============================
```

Comment:
The header line needs to be longer than the title: Not to be a pain in the arse, but sphinx-gallery is going to fail on this :D

Reply:
Thanks for letting me know :) . I am not familiar with sphinx, will fix, thanks for pointing this out :) .